rest/config_legacy.go

```diff
@@ -427,7 +427,7 @@ func ParseCommandLine(args []string, handling flag.ErrorHandling) (*LegacyServer
 	_ = flagSet.Bool("api.admin_interface_authentication", true, "")
 	_ = flagSet.Bool("api.metrics_interface_authentication", true, "")
 	_ = flagSet.Bool("bootstrap.allow_insecure_server_connections", false, "")
-	_ = flagSet.Bool("bootstrap.allow_insecure_tls_connections", false, "")
+	_ = flagSet.Bool("api.https.allow_insecure_tls_connections", false, "")
```

Comment:
I think this is already done, which means the PR should be rebased onto the latest master.